openshift/settings.sh

```diff
@@ -1,3 +1,3 @@
 export PROJECT_NAMESPACE="a99fd4"
-export GIT_URI="https://github.com/bcgov/covid-19-essential-services.git"
+export GIT_URI="https://github.com/bcgov/essential-services.delivery.git"
```

Comment:
The repository name needs fixing, it should be `essential-services-delivery`.

Reply:
changes completed